### ISSUE TITLE
database.createDatabase cannot define the access level for newly created users

### ISSUE THREAD

**Issue by rcoulsell** [arangodb/arangojs#664]
When creating a new database and defining users using the database.createDatabase users: Array<Object>, it creates a new user with administative rights to the subject database.

This is causing difficulty in automatically setting up default system users for app integration/installation in Node.js.

Is there a way to pass in a parameter under the 'extra' object to set the access for the user to the created database as 'Access' instead of 'Adminstrate'?

Also, is there a way to pass in a parameter under the 'extra' object to set the default access for the user as well?

**Comment by rcoulsell** [comment]
I know this can be achieved using an external curl call outside of arangojs after the fact:
curl -X PUT -u "<admin user>:<admin user password>" -H "Content-Type: application/json" -H "accept: application/json" -d '{"grant" : "ro"}' http://localhost:8529/_api/user/{userModified}/database/{databaseModified}

Seems like this is missing from arangojs natively.

**Comment by pluma** [comment]
No, this option is intended to create per-database admin accounts as an alternative to the default `root` user. This option pre-dates the modern permission system, so this may be an oversight in the API: https://www.arangodb.com/docs/stable/http/database-database-management.html#create-database

You're right that there is currently no way to manage users from arangojs. You can however access arbitrary endpoints of the HTTP API using the `.route()` method, which provides a generic request API.

**Comment by pluma** [comment]
I think this may be a duplicate of https://github.com/arangodb/arangojs/issues/593

**Comment by pluma** [comment]
The HTTP API is documented at https://www.arangodb.com/docs/stable/http/user-management.html
